sdcm/send_email.py

```diff
@@ -385,9 +385,37 @@ class PrivateRepoEmailReporter(BaseEmailReporter):
     email_template_file = "results_private_repo.html"
 
 
+class TestAbortedEmailReporter(BaseEmailReporter):
+    _fields = ("repo_ostype",
+               "repo_uuid",
+               "scylla_repo",)
+    email_template_file = "results_private_repo.html"
```

Comment:
May be better to rename to test_aborted_email.html?

Reply:
Removed it, it is not used there.